src/openms/include/OpenMS/KERNEL/MSExperiment.h

```diff
@@ -537,14 +537,15 @@ namespace OpenMS
     //@}
 
     /**
-    @brief Compute Total Ion Count per MS1 spectrum and applies the resampling algorithm, if a bin size in RT seconds greater than 0 is given.
+    @brief Compute Total Ion Count per MS spectrum and applies the resampling algorithm, if a bin size in RT seconds greater than 0 is given.
 
-    By default, each MS1 spectrum's intensity just gets summed up. Regular RT bins can be obtained by specifying @p rt_bin_size.
+    By default, each MS spectrum's intensity just gets summed up. Regular RT bins can be obtained by specifying @p rt_bin_size.
 
     @param bin_size RT bin size in seconds
+    @param ms_level MS level of spectra for calculation
     @return TIC Chromatogram
     **/
-    const MSChromatogram getTIC(float rt_bin_size = 0) const;
+    const MSChromatogram getTIC(float rt_bin_size = 0, UInt ms_level = 1) const;
```

Comment:
sounds good.
Is there a CV Term or some kind of annotation for these chromatograms?
If yes, the above code could also look for them instead of recomputing the TIC ... and then the name would not be too bad.